src/components/molecules/AdvancedLendingTopBar/AdvancedLendingTopBar.stories.tsx

```diff
@@ -10,6 +11,7 @@ const lastTradePrice = 8000;
 export default {
     title: 'Molecules/AdvancedLendingTopBar',
     component: AdvancedLendingTopBar,
+    decorators: [withWalletProvider],
```

Comment:
is a wallet connection required on AdvancedLendingTopBar?

Reply:
@as472 Yes for the story, because I am using retrieving wallet address is CurrencyMaturityDropdown, one of its child components.